### PR TITLE
docs: update Kinsta deployment documentation

### PR DESCRIPTION
**Pull request by palmiak**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation

At Kinsta, we just added a new service that will fit Docusarus better than the previously mentioned in the documentation.

### Test links

Deploy preview: https://deploy-preview-9430--docusaurus-2.netlify.app/

## Related issues/PRs

https://github.com/facebook/docusaurus/pull/8375 - initial PR for adding Kinsta

### CODE REVIEW

File: website/docs/deployment.mdx [hunk 1]
```diff
@@ -830,6 +830,7 @@ See [docs](https://docs.quantcdn.io/docs/cli/continuous-integration) and [blog](
...
 
-## Deploying to Kinsta Application Hosting {#deploying-to-kinsta}
+## Deploying to Kinsta {#deploying-to-kinsta}
 
-[Kinsta Application Hosting](https://kinsta.com/application-hosting) is a service that lets you build and deploy your web apps directly from your Git repository. Get started in just a minutes by following our [Docusaurus on Kinsta](https://kinsta.com/help/docusaurus-example/) article.
+[Kinsta Static Site Hosting](https://kinsta.com/static-site-hosting) lets you deploy up to 100 static sites for free, custom domains with SSL, 100 GB monthly bandwidth, and 260+ Cloudflare CDN locations.
+Get started in just a few clicks by following our [Docusaurus on Kinsta](https://kinsta.com/docs/docusaurus-example/) article.
```
Comment (slorber) [inline website/docs/deployment.mdx:836]:
```suggestion
[Kinsta Static Site Hosting](https://kinsta.com/static-site-hosting) lets you deploy up to 100 static sites for free, custom domains with SSL, 100 GB monthly bandwidth, and 260+ Cloudflare CDN locations.

Get started in just a few clicks by following our [Docusaurus on Kinsta](https://kinsta.com/docs/docusaurus-example/) article.
```

**Review by slorber — APPROVED** [review]
thanks 👍

**Comment by palmiak** [comment]
Thanks. 

BTW - one question: why does Kinsta appear only in the canary version, not in the main one?
https://docusaurus.io/docs/deployment - Azure is the last one
https://docusaurus.io/docs/next/deployment - Kinsta is listed at the end

**Comment by slorber** [comment]
> Thanks.
> 
> BTW - one question: why does Kinsta appear only in the canary version, not in the main one? [docusaurus.io/docs/deployment](https://docusaurus.io/docs/deployment) - Azure is the last one [docusaurus.io/docs/next/deployment](https://docusaurus.io/docs/next/deployment) - Kinsta is listed at the end

because you didn't backport your changes to older versioned docs

But it doesn't matter because v3.0 is going to be released very soon and it will be on /docs/deployment very soon

**Comment by slorber** [comment]
FYI, we are going to clean up our existing Docusaurus docs deployment page in favor of a GitHub discussions category. 

This means your platform won't appear anymore on this page: https://docusaurus.io/docs/deployment

If you want your hosting / CI / CDN platform to remain documented for Docusaurus users, please submit an entry at: https://github.com/facebook/docusaurus/discussions/categories/deployment-platforms

This will be your responsibility to keep it up to date over time.

**Pull request merged**